metrics_block.py

```diff
@@ -54,6 +65,56 @@ def stop(self):
         if self._metrics_job is not None:
             self._metrics_job.cancel()
 
+    def timestamp(self):
+        '''returns the current system timestamp'''
+        return datetime.isoformat(datetime.utcnow())
+
+    @staticmethod
```

Comment:
No,  it was because it never used a self.  I can definitely remove it,  but
it is a valid point!
On Feb 16, 2015 12:45 PM, "Matt Dodge" <notifications@github.com> wrote:

> In metrics_block.py
> <https://github.com/nio-blocks/metrics/pull/2#discussion_r24771681>:
>
> > @@ -54,6 +65,56 @@ def stop(self):
> >          if self._metrics_job is not None:
> >              self._metrics_job.cancel()
> >
> > +    def timestamp(self):
> > +        '''returns the current system timestamp'''
> > +        return datetime.isoformat(datetime.utcnow())
> > +
> > +    @staticmethod
>
> Ohhh it's because it doesn't have a self in the arguments list, that's why
> your editor is throwing that at you.
>
> I think add the self remove the staticmethod
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nio-blocks/metrics/pull/2/files#r24771681>.
>